components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/PreferenceRetrievalClient.java

```diff
@@ -98,6 +99,11 @@ public boolean checkSelfRegistrationLockOnCreation(String tenant) throws Prefere
         return checkPreference(tenant, SELF_SIGN_UP_CONNECTOR, SELF_SIGN_UP_LOCK_ON_CREATION_PROPERTY);
     }
 
+    public boolean checkSelfRegistrationSendConfirmationOnCreation(String tenant) throws PreferenceRetrievalClientException {
```

Comment:
add the method comment

Reply:
Addressed with commit f760e93122e5048275de82590980f8df9b91b51e